app/verify/verify-content.js

```diff
@@ -0,0 +1,22 @@
+const crypto = require('crypto')
+
+const verifyContent = (content, hash) => {
+  try {
+    const fileHash = crypto.createHash('sha256').update(content).digest('hex')
+    const isValid = fileHash === hash
+
+    if (isValid) {
+      console.log('File content successfully verified')
+    } else {
+      console.log('File verification failed')
+      console.log(`File hash: ${fileHash}`)
+      console.log(`Validation hash: ${hash}`)
+    }
+    return isValid
```

Comment:
I would explicitly set this to `true` so the return type is obvious

Reply:
> Could you update the diagrams repo with an alternative diagram with this in-place of Function App?

I feel we need some grand unveiling of the icon change!

Yes I have done this, I've just not pushed it yet until it's true.

Reply:
until the grand revealing you mean